File: .py/1822.SignofAProductofAnArray.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        n = len(nums)
        product = 1
        for i in range(n):
            if nums[i]==0:
                return 0
            elif nums[i]<0:
                product *= -1

        return product
        

File: .cpp/1822.SignofAProductofAnArray.cpp
class Solution {
public:
    
    int arraySign(vector<int>& nums) {
        int n = nums.size();
        int product = 1;
        for (int i=0;i<n;i++)
        {
            if(nums[i]==0)
            {
                return 0;
            }
            else if(nums[i]<0)
            {
                product *=-1;
            }
        }
        return product;

    }
};